docs/docusaurus/docs/cloud/alerts/trigger_actions.md

```diff
@@ -43,18 +43,16 @@ Use Actions to notify the appropriate parties of the results of your Validation
    ```
 
    :::tip The GX-managed Checkpoint name can be found through the UI
-   For the Data Asset of interest, go to the **Validations** tab, select the appropriate **Expectation Suite**, click the code snippet icon next to the **Validate** button, and then click **Generate snippet**.
+   For the Data Asset of interest, go to the **Validations** tab. If you have more than one **Expectation Suite**, select the **GX-managed** one. Then, click the code snippet icon next to the **Validate** button, and then click **Generate snippet**.
 
 3. Define the Actions that the Checkpoint will trigger.
     
-   The following is an example of how to append a [`SlackNotificationAction`](../../reference/api/checkpoint/SlackNotificationAction_class) to the Action list of your Checkpoint.
+   The following is an example of how to define a [`SlackNotificationAction`](/reference/api/checkpoint/SlackNotificationAction_class.mdx) to the Action list of your Checkpoint.
```

Comment:
Thanks for changing append to define. There's some lingering text that I suggested removing in the last round of feedback that I still think doesn't fit the scope of this step. I also don't think it makes sense with the changed verb. 

```suggestion
   The following is an example of how to define a [`SlackNotificationAction`](/reference/api/checkpoint/SlackNotificationAction_class.mdx).
```